components/service-operator/controllers/principal_cloudformation.go

```diff
@@ -24,6 +24,12 @@ func PrincipalCloudFormationController(c sdk.Client) Controller {
 				ParameterKey:   aws.String(access.IAMPermissionsBoundaryParameterName),
 				ParameterValue: aws.String(env.AWSPrincipalPermissionsBoundaryARN()),
 			},
+			{
+				ParameterKey:   aws.String(access.ServiceOperatorIAMRoleArn),
+				ParameterValue: aws.String(env.AWSRoleArn()),
```

Comment:
```suggestion
				ParameterValue: aws.String(env.GetAWSRoleArn()),
```

Reply:
actually ... looks like it's the one with "Get" that was the odd one out... so I changed that one so they are all consistant